user/common-build-problems.md

```diff
@@ -576,3 +576,36 @@ When publishing via the `nexus-staging-maven-plugin` to Sonatype OSS Repository,
 ## Travis CLI does not recognize my valid Github Token
 
 When using the [Travis CLI tool](https://github.com/travis-ci/travis.rb#readme) to interact with the Travis CI platform, if you receive an `insufficient_oauth_permissions` error or similar, please ensure the Github Token supplied via `--github-token` has **repo** scope as [explained in this document](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/).
+
+## Duplicate/Unknown Job shows up in my build
+
+When specifying stages, users often unknowingly add an implicit _Job_ to the list of _Jobs_ in a _Stage_ due to a syntactically correct YAML entry. For example:
```

Comment:
This is slightly confusing to me. It isn't the fact that the yaml is correct that is causing the job, but the fact that it doesn't do what they think it does *despite* being syntactically correct.

I'm not quite sure how to generalize that though. Can we say something about the `-` here?

Reply:
We don't usually `_jobs_` or `_stages_`. Even if it would make sense to, lets not introduce it without consistency here.

Reply:
Will update. Thanks